man/systemd.netdev.xml

```diff
@@ -1975,6 +1978,107 @@
+        <term><varname>Fragmentation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables fragmentation. Defaults to true.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>HopPenalty=</varname></term>
+        <listitem>
+          <para>The hop penalty setting allows to modify
+          <citerefentry><refentrytitle>batctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+          preference for multihop routes vs. short routes. This interger value is applied to the
+          TQ (Transmit Quality) of each forwarded OGM (Originator Message), thereby propagating the
+          cost of an extra hop (the packet has to be received and retransmitted which costs airtime).
+          A higher hop penalty will make it more unlikely that other nodes will choose this node as
+          intermediate hop towards any given destination. The default hop penalty of '15' is a reasonable
+          value for most setups and probably does not need to be changed. However, mobile nodes could
+          choose a value of 255 (maximum value) to avoid being chosen as a router by other nodes.
+          The minimum value is 0.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>OriginatorIntervalSec=</varname></term>
+        <listitem>
+          <para>The value specifies the interval in seconds, unless another time unit is specified in which
+          batman-adv floods the network with its protocol information.
+          See <citerefentry><refentrytitle>systemd.timed</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+          for more information.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>GatewayBandwidthDown=</varname></term>
+        <listitem>
+          <para>If the node is a server, this
+          parameter is used to inform other nodes in the network about
+          this node's internet connection download bandwidth in bits per second. Just enter any number
+          suffixed with K, M, G or T and the batman-adv
+          module will propagate the entered value in the mesh.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>GatewayBandwidthUp=</varname></term>
+        <listitem>
+          <para>If the node is a server, this
+          parameter is used to inform other nodes in the network about
+          this node's internet connection upload bandwidth in bits per second. Just enter any number
+          suffixed with K, M, G or T and the batman-adv
+          module will propagate the entered value in the mesh.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>RoutingAlgorithm=</varname></term>
+        <listitem>
+          <para>This can be either <literal>BATMAN_V</literal> or <literal>BATMAN_IV</literal> and describes which routing_algo
```

Comment:
We usually use lowercase with hyphen. So, maybe `batman-v` or `batman-5`? Moreover, `BATMAN_` prefix sounds redundant. So, maybe simply `4` or `5` are ok?